<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-03-14T20:20:45 IST" name="Suite" finished-at="2023-03-14T20:20:51 IST" duration-ms="6059">
    <groups>
    </groups>
    <test started-at="2023-03-14T20:20:45 IST" name="Test" finished-at="2023-03-14T20:20:51 IST" duration-ms="6059">
      <class name="com.CMS.TestCases.ServiceModuleTestCases">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.CMS.TestCases.ServiceModuleTestCases@2631f68c]" started-at="2023-03-14T20:20:45 IST" name="setup" finished-at="2023-03-14T20:20:46 IST" duration-ms="1449" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="beforeRun()[pri:0, instance:com.CMS.TestCases.ServiceModuleTestCases@2631f68c]" started-at="2023-03-14T20:20:46 IST" name="beforeRun" finished-at="2023-03-14T20:20:48 IST" duration-ms="1563" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeRun -->
        <test-method signature="validateChangePasswordFunction()[pri:0, instance:com.CMS.TestCases.ServiceModuleTestCases@2631f68c]" started-at="2023-03-14T20:20:48 IST" name="validateChangePasswordFunction" finished-at="2023-03-14T20:20:49 IST" duration-ms="1512" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateChangePasswordFunction -->
        <test-method signature="verifyNewPasswordField(java.lang.String,java.lang.String)[pri:1, instance:com.CMS.TestCases.ServiceModuleTestCases@2631f68c]" started-at="2023-03-14T20:20:50 IST" name="verifyNewPasswordField" data-provider="testData_Password" finished-at="2023-03-14T20:20:50 IST" duration-ms="240" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Abcd123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Abcdefgh]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.CMS.PageObjects.ServiceModule.newPasswordFieldCheck(ServiceModule.java:78)
at com.CMS.TestCases.ServiceModuleTestCases.verifyNewPasswordField(ServiceModuleTestCases.java:48)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyNewPasswordField -->
        <test-method is-config="true" signature="afterRun()[pri:0, instance:com.CMS.TestCases.ServiceModuleTestCases@2631f68c]" started-at="2023-03-14T20:20:51 IST" name="afterRun" finished-at="2023-03-14T20:20:51 IST" duration-ms="133" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterRun -->
      </class> <!-- com.CMS.TestCases.ServiceModuleTestCases -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
